tasks/verify_translations.js

```diff
@@ -17,11 +17,13 @@
  * under the License.
  */
 
+// TODO: Integrate a new tool for translations checking
```

Comment:
Before merging this in, I think we should either resolve the TODO comments or else create an issue and drop the link here, so we don't lose track.